zerver/lib/events.py

```diff
@@ -923,6 +923,10 @@ def fetch_initial_state_data(
         # abuse.
         state["giphy_api_key"] = settings.GIPHY_API_KEY if settings.GIPHY_API_KEY else ""
 
+    if want("tenor"):
```

Comment:
Probably should have a comment here saying "See Giphy comment above; Tenor API keys work similarly,"